src/index.js

```diff
@@ -3,16 +3,10 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
-import reportWebVitals from './reportWebVitals';
```

Comment:
- [ ] Well done removing line 6, but you have not removed the files and  there are still boilerplate created by the CRA(texts, images, styles) in `public `and `src `folders which you are not going to use. Kindly delete them.